mlflow/ml-package-versions.yml

```diff
@@ -645,6 +645,7 @@ langchain:
           "numexpr",
           "hf_transfer",
         ]
+      "== 0.1.14": ["langchain-core<=0.1.37"]
```

Comment:
https://github.com/langchain-ai/langchain/blob/v0.1.14/libs/langchain/pyproject.toml#L15 indicates that 0.1.37 should work.

```
langchain-core = "^0.1.37"
```

Reply:
`^0.1.37` is equivalent to `< 0.2.0` and `>= 0.1.37`.